tweetapi/login.py

```diff
@@ -0,0 +1,76 @@
+from django.conf import settings
+import jwt
+from .serializer import UserSerializer
+from .registration import UserloginSerializer
+from rest_framework.generics import CreateAPIView
+import datetime
+from django.contrib.auth.models import User
+from django.contrib.auth import logout,authenticate
+from rest_framework.views import APIView
+from rest_framework import status
+from rest_framework.response import Response
+from django.conf import settings
+
+
+
+class Login(CreateAPIView):
+    serializer_class = UserloginSerializer
+    def post(self, request,format=None):
+        serializer = UserloginSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
```

Comment:
this is not how to use `is_valid` and `raise_exception`

Reply:
i was trying to use is_valid() method to perform validation for each field, raise_exception=True flag, i was looking to override the style of validation error responses, how can i use it, and am i to use the is_valid() without the raise_exception and hows the best way to use it sir

Reply:
you can't check is valid and still raise exceptions at the same time. pick one

Reply:
`serializer.is_valid(raise_exception=True)`

or

`if serializer.is_valid():`

Reply:
fixed sir